app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function `copy_file` correctly takes a single string argument `command`, as required by the task description.